sdk/cosmos/azure-cosmos/test/test_user_configs.py

```diff
@@ -46,5 +58,54 @@ def test_enable_endpoint_discovery(self):
         self.assertTrue(client_default.client_connection.connection_policy.EnableEndpointDiscovery)
         self.assertTrue(client_true.client_connection.connection_policy.EnableEndpointDiscovery)
 
+    def test_default_account_consistency(self):
```

Comment:
Do we have similar test in async?

Reply:
async tests for the python sdk will come altogether in a later PR, we dont currently have any